src/common/components/clusterConfiguration/networkConfigurationValidation.ts

```diff
@@ -9,33 +9,12 @@ import {
   vipValidationSchema,
 } from '../ui';
 
-import { getSubnetFromMachineNetworkCidr, getHostSubnets } from './utils';
```

Comment:
code change in common - since code is copied might as well leave as is

Reply:
Reverted. Thanks for spotting it.